interview-app/src/main/java/ua/edu/ukma/interviewapp/InterviewApplication.java

```diff
@@ -0,0 +1,13 @@
+package ua.edu.ukma.interviewapp;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class InterviewApplication {
+
+    public static void main(String[] args) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.
<!-- __reviewdog__:ChBjOGJkYmE1MTg0MGZmNGFkEgpjaGVja3N0eWxl -->